boa_engine/src/vm/code_block.rs

```diff
@@ -110,7 +164,8 @@ pub struct CodeBlock {
     pub(crate) params: FormalParameterList,
 
     /// Bytecode
-    pub(crate) bytecode: Box<[u8]>,
+    #[unsafe_ignore_trace]
+    pub(crate) bytecode: Box<[Cell<u8>]>,
```

Comment:
As far as I can see the `Cell` is not needed here anymore right?

Reply:
Yes, right now it's not needed, but I kept for implementing lazy inline cache entry generation.

I'll remove it and add it when I create the PR :)

Reply:
I did wonder if using a Vec<u8> would be more efficent in this case, since `UnsafeCell` prevents optimizations. Probably would be good to benchmark explicitly for the bytecode field.

Reply:
I don't think it would change much if it was a `Vec` since we would still have to have some kind of interior mutability (`Vec<Cell<u8>>`) or wrapping the Vec into a `RefCell` (but this requires keeping borrow state, which I think would be less efficient, but allowing to shrink/grow which for the lazy inline catching I don't need).